src/api/custom-resolvers/mutations/delete-application-by-reference-number/index.test.ts

```diff
@@ -0,0 +1,50 @@
+import deleteApplicationByReferenceNumber from '.';
+import applications from '../../../test-helpers/applications';
+import { Application, SuccessResponse } from '../../../types';
+import getKeystoneContext from '../../../test-helpers/get-keystone-context';
+
+const context = getKeystoneContext();
+
+describe('custom-resolvers/delete-application-by-reference-number', () => {
```

Comment:
missing /mutations.  not a blocker

Reply:
Good spot, will fix in next PR so we can merge this in and avoid flaky tests in the PR backlog